chromium_src/extensions/browser/verified_contents.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_EXTENSIONS_BROWSER_VERIFIED_CONTENTS_H_
+#define BRAVE_CHROMIUM_SRC_EXTENSIONS_BROWSER_VERIFIED_CONTENTS_H_
+
+#define Create(...)                 \
```

Comment:
where does this actually get called?

Reply:
content_hash.cc?

Reply:
or SandboxedUnpacker? Both

Reply:
since we know these are signed by the brave key and must be signed by the brave key I think it makes more sense to subclass `CrxInstaller` and override `GetContentVerifierKey` and use that instead of `CrxInstaller::CreateSilent`

Reply:
Both

Reply:
Overriding `GetContentVerifierKey` will only allow us to check the contents on install.

Reply:
actually I think this can be done without patching by doing `#define VerifiedContents BraveVerifiedContents` in content_hash.cc and sandboxed_unpacker.cc

put BraveVerifiedContents directly in verified_contents.h/verified_contents.cc chromium_src overrides

it will not be a subclass, you only need to define relevant static methods for it